demos/smartlab_demo/python/display.py

```diff
@@ -16,81 +13,200 @@
 
 import numpy as np
 import cv2
+import os
 
 
 class Display:
     def __init__(self):
         '''Score Evaluation Variables'''
-
-    def display_result(self, frame_top, frame_front, front_seg_results, top_seg_results, top_det_results, front_det_results, scoring, state, frame_counter, fps):
-        #renew score board so that when put cv2.puttext text will not overlap
-        self.score_board = np.zeros([200, 1920, 3], dtype=np.uint8)
-
-        #add action name of each frame at middle top
-        cv2.putText(frame_top, front_seg_results, (700, 80), cv2.FONT_HERSHEY_SIMPLEX, color=(0, 0, 255), fontScale=1.5, thickness=3)
-        cv2.putText(frame_front, top_seg_results, (700, 80), cv2.FONT_HERSHEY_SIMPLEX, color=(0, 0, 255), fontScale=1.5, thickness=3)
-
-        #display frame_number at top left corner
-        cv2.putText(frame_front, f"frame{frame_counter: 6d}", (50, 80), cv2.FONT_HERSHEY_SIMPLEX, color=(0, 0, 255), fontScale=1.5, thickness=3)
-
-        #display FPS at top left corner
-        cv2.putText(frame_top, f"FPS: {fps: .2f}", (50, 160), cv2.FONT_HERSHEY_SIMPLEX, color=(0, 0, 255), fontScale=1.5, thickness=3)
-        cv2.putText(frame_front, f"FPS: {fps: .2f}", (50, 160), cv2.FONT_HERSHEY_SIMPLEX, color=(0, 0, 255), fontScale=1.5, thickness=3)
-
-        # show current state for troubleshooting purpose
-        cv2.putText(frame_top, state, (1500, 80), cv2.FONT_HERSHEY_SIMPLEX, color=(0, 0, 255), fontScale=1.5, thickness=3)
-        cv2.putText(frame_front, state, (1500, 80), cv2.FONT_HERSHEY_SIMPLEX, color=(0, 0, 255), fontScale=1.5, thickness=3)
-
-        #display obj detection result for both view
-        for row, obj_cls in zip(top_det_results[0], top_det_results[1]):
-            x_min = int(row[0])
-            y_min = int(row[1])
-            x_max = int(row[2])
-            y_max = int(row[3])
-
-            cv2.putText(frame_top, obj_cls, (x_min, y_min-10), cv2.FONT_HERSHEY_SIMPLEX, color=(0, 0, 255), fontScale=0.9, thickness=2)
-            frame_top = cv2.rectangle(frame_top, (x_min, y_min), (x_max, y_max), color=(255, 0, 0), thickness=2)
-
-        for row, obj_cls in zip(front_det_results[0], front_det_results[1]):
-            x_min = int(row[0])
-            y_min = int(row[1])
-            x_max = int(row[2])
-            y_max = int(row[3])
-
-            cv2.putText(frame_front, obj_cls, (x_min, y_min-10), cv2.FONT_HERSHEY_SIMPLEX, color=(0, 0, 255), fontScale=0.9, thickness=2)
-            frame_front = cv2.rectangle(frame_front, (x_min, y_min), (x_max, y_max), color=(255, 0, 0), thickness=2)
+        self.wait_icon = cv2.imread(f'{os.getcwd()}/icon/wait.png', cv2.IMREAD_UNCHANGED)
+        self.no_icon = cv2.imread(f'{os.getcwd()}/icon/no.png', cv2.IMREAD_UNCHANGED)
+        self.done_icon = cv2.imread(f'{os.getcwd()}/icon/done.png', cv2.IMREAD_UNCHANGED)
```

Comment:
This relies on current working dir which can be anything. You must use `__file__` instead.